<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>640</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>IBeamCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>871</width>
      <height>511</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>855</width>
       <height>495</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>1</y>
        <width>851</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/my_images/icons/I.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew_N"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_P"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;Eï¼‰</string>
    </property>
    <addaction name="actionCut_X"/>
    <addaction name="actionCopy_C"/>
    <addaction name="actionPaste_P"/>
    <addaction name="actionUndo_Z"/>
    <addaction name="actionRedo_Y"/>
    <addaction name="separator"/>
    <addaction name="actionFind_F"/>
    <addaction name="actionSelect_All_A"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionDocs"/>
   </widget>
   <widget class="QMenu" name="menuAbout_A">
    <property name="title">
     <string>About(&amp;A)</string>
    </property>
    <addaction name="actionMiniword"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuHelp_H"/>
   <addaction name="menuAbout_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_N"/>
   <addaction name="actionOpen_O"/>
   <addaction name="actionSave_S"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionPrint_P"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionCut_X"/>
   <addaction name="actionCopy_C"/>
   <addaction name="actionPaste_P"/>
   <addaction name="actionUndo_Z"/>
   <addaction name="actionRedo_Y"/>
   <addaction name="actionFind_F"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_All_A"/>
   <addaction name="actionDocs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_N">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/new.png</normaloff>:/my_images/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/open.png</normaloff>:/my_images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/Save.png</normaloff>:/my_images/icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/saveas.png</normaloff>:/my_images/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint_P">
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/exit.png</normaloff>:/my_images/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut_X">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/cut.png</normaloff>:/my_images/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_C">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/copy.png</normaloff>:/my_images/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/paste.png</normaloff>:/my_images/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo_Z">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/undo_256.png</normaloff>:/my_images/icons/undo_256.png</iconset>
   </property>
   <property name="text">
    <string>Undo(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo_Y">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/redo.png</normaloff>:/my_images/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo(&amp;Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionFind_F">
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSelect_All_A">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/changefont.png</normaloff>:/my_images/icons/changefont.png</iconset>
   </property>
   <property name="text">
    <string>Select All(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/my_images/icons/aboutqt.png</normaloff>:/my_images/icons/aboutqt.png</iconset>
   </property>
   <property name="text">
    <string>Docs</string>
   </property>
  </action>
  <action name="actionMiniword">
   <property name="text">
    <string>Miniword</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
